ocp_resources/resource.py

```diff
@@ -134,9 +133,27 @@ def sub_resource_level(current_class: Any, owner_class: Any, parent_class: Any)
     return None
 
 
-# Exceptions classes
+def change_dict_value_to_hashed(resource_dict: Dict[Any, Any], key_name: str) -> Dict[Any, Any]:
```

Comment:
find a better name for it?

- `mask_key_values`
- `replace_key_with_hashed_value`
- `hide_key_in_dict`

Reply:
Done.